conf/config.json

```diff
@@ -21,6 +21,9 @@
         }
     },
     "extensions": {
+        "lifecycle": {
```

Comment:
I think duplicate from pre-rebase

Reply:
Indeed, @bennettbuchanan can you fix when you have a chance?

Reply:
Thanks for this @philipyoo. I've updated it.